src/backend/chat_service/chat_service.py

```diff
@@ -133,6 +133,10 @@ async def upload_File(request: Request, customer_guid: str = Form(...), file:Upl
 async def list_files(request: Request, customer_guid: str):
     logger.debug(f"Entering list_files() with Correlation ID: {request.state.correlation_id}")
     try:
+        if not db_manager.check_customer_guid_exists(customer_guid):
```

Comment:
Instead of checking whether customer_guid exists here, you can check whether customer_guid exists in minio_manager by verifying whether customer_guid bucket exists.

If it does not exist you can throw an exception saying "customer_guid" not found with the correct HTTP ERROR CODE

Remove these lines
        if not db_manager.check_customer_guid_exists(customer_guid):
            logger.error(f"Invalid customer_guid: {customer_guid}")
            raise HTTPException(status_code=404, detail="Invalid customer_guid provided")